### ISSUE TITLE
Prefer TLS connections in documentation

### ISSUE THREAD

**Issue by haphut** [neondatabase/neon#2148]
The current Neon onboarding starts by a suggestion to import my data. Yet as far as I can see, all of the current Neon documentation guides developers to use unencrypted connections.

I see from the Certificate Transparency logs that you already have a Let's Encrypt wildcard certificate for `*.cloud.neon.tech`.

I think it makes sense to have secure defaults. Please switch the `psql -h pg.neon.tech` command and all of the example connection strings in the documentation and in the code samples to use TLS.

Great to see Neon take shape! You're building an exciting service.

**Comment by kelvich** [comment]
Makes sense. So far, password flow (`<project>.cloud.neon.tech`) is using only TLS -- even if you try to connect without TLS our proxy will not allow this. But `pg.neon.tech` is a bit different and configured without TLS now. Will fix
